Automattic/sensei
# Enable to show the course category

### Changes proposed in this Pull Request
* Add the Course Category block
* Add a factory to create course-categories

### Testing instructions
* Create a page
* Add a Course List block (please pay attention if the post-type is equal to "course" 
* Add the course categories block.
* Save
* Check if the block is rendered
* Check the acceptance criteria described on the card.
